gulpfile.babel.js

```diff
@@ -153,6 +157,7 @@ function watch() {
   gulp.watch('src/pages/**/*.html').on('all', gulp.series(pages, browser.reload));
   gulp.watch('src/{layouts,partials}/**/*.html').on('all', gulp.series(resetPages, pages, browser.reload));
   gulp.watch('src/assets/scss/**/*.scss').on('all', sass);
+  gulp.watch(['src/assets/scss/**/*{tff,woff,woff2,eot,svg}.']['fonts']);
```

Comment:
You're watching `scss` there, but same remark as above: rely on `PATH.fonts`.

Reply:
Can you elaborate a bit on what you mean by `PATH.fonts`?

Thanks!

Reply:
Just saw your other comment. It was hidden thanks to a commit I made just a few minutes ago.